docs/central-install-digital-ocean.rst

```diff
@@ -296,21 +296,21 @@ Central ships with a basic EXIM server bundled to forward mail out to the intern
 1. Ensure you have an SMTP relay server visible to your Central server network host.
 2. Edit the file ``files/service/config.json.template`` to reflect your network hostname, the TCP port, and authentication details. The ``secure`` flag is for TLS and should be set to ``true`` if the port is 465 and ``false`` for other ports. If no authentication is required, remove the ``auth`` section.
 
-  .. code-block:: console
-
-   "email": {
-     "serviceAccount": "my-replyto-email",
-     "transport": "smtp",
-     "transportOpts": {
-       "host": "smtp.example.com",
-       "port": 587,
-       "secure": false,
-       "auth": {
-         "user": "my-smtp-user",
-         "pass": "my-smtp-password"
+   .. code-block:: json
+
+     "email": {
+       "serviceAccount": "my-replyto-email",
+       "transport": "smtp",
+       "transportOpts": {
+         "host": "smtp.example.com",
+         "port": 587,
+         "secure": false,
+         "auth": {
+           "user": "my-smtp-user",
+           "pass": "my-smtp-password"
+         }
        }
      }
-   }
 
 3. Build and run: ``docker-compose build service``, ``docker-compose stop service``, ``docker-compose up -d service``.
```

Comment:
I'm seeing `docker-compose` in the digital ocean install instructions, e.g. in Advanced Configuration Options and earlier in Changing Server Settings.

Reply:
Should now be fixed.